src/main/java/cash/z/ecc/android/sdk/transaction/TransactionRepository.kt

```diff
@@ -76,6 +77,8 @@ interface TransactionRepository {
 
     suspend fun count(): Int
 
+    suspend fun getAccount(accountId: Int): UnifiedAddressAccount?
```

Comment:
This feels out of place. I'd add the AccountRepository to the CompactBlockProcessor or SDKSynchronizer directly rather than adding this function

Reply:
Good callout this certainly needs to be refactored. I've spun that out into a separate github issue: https://github.com/zcash/zcash-android-wallet-sdk/issues/220